x-pack/plugins/security_solution/public/common/components/charts/donutchart.tsx

```diff
@@ -48,7 +50,11 @@ export interface DonutChartProps {
   height?: number;
   label: React.ReactElement | string;
   legendItems?: LegendItem[] | null | undefined;
-  onElementClick?: ElementClickListener;
+  /**
+   * provides the section name of a clicked donut ring partition
+   */
+  onDonutPartitionClicked?: (level: string) => void;
```

Comment:
NIT: we are in the donut component, so there's no need to include the word "donut" in the prop names, it is implicit in all the props.
Also, we usually use "click" instead of "clicked" for click events, so, to be consistent, it should be `onPartitionClick` (and `onLabelClick` in the next line).

Reply:
fixed, thank you